### ISSUE TITLE
which-key-add-keymap-based-replacements for prefix

### ISSUE THREAD

**Issue by Ergus** [justbur/emacs-which-key#316]
Hi:

I use the command  which-key-add-keymap-based-replacements to substitute the +prefix entries when possible. 

Up to now I have something like `(which-key-add-keymap-based-replacements flyspell-mode-map "C-c f" "flyspell")` but I see that the new syntax wants a const pair there as the last argument...

What's the intended way to specify that? use a nil?

**Comment by justbur** [comment]
It should actually accept what you have there. Is that line not working?

**Comment by Ergus** [comment]
> It should actually accept what you have there. Is that line not working?

Yes, it works, but it is not a cons pair. That is the recommended way to do it according to the documentation.

**Comment by justbur** [comment]
The cons pair is more explicit. The string version is for backwards compatibility. With a string, the function constructs the cons pair `(STRING . BINDING)` by looking up the corresponding key binding.
